### PR TITLE
ExperimentData.store documentation is not correct

### LINKED ISSUE #190: ExperimentData.store documentation is not correct

### PR DESCRIPTION
**Pull request by mpvanderschelling**
Fixes #190

### CODE REVIEW

**Comment by mpvanderschelling** [comment]
Also I changed the order of the ExperimentSample.store() method so that the name is the first argument.
This resembles other methods where a name is provided i.e. Domain.add_float(name, low, high)

File: src/f3dasm/_src/experimentdata/experimentsample.py [hunk 1]
```diff
@@ -349,27 +349,26 @@ def to_dict(self) -> Dict[str, Any]:
         """
         return {**self.input_data, **self.output_data_loaded, 'job_number': self.job_number}
 
-    def store(self, object: Any, name: str, to_disk: bool = False,
+    def store(self, name: str, object: Any, to_disk: bool = False,
```
Comment (BernardoFerreira) [inline src/f3dasm/_src/experimentdata/experimentsample.py:352]: Please clarify in the method docstring where the object is actually stored.

As far as I can understand from the documentation, the user is only providing the basename of the storage file.

File: src/f3dasm/_src/experimentdata/experimentdata.py [hunk 2]
```diff
@@ -298,7 +298,7 @@ def select(self, indices: int | slice | Iterable[int]) -> ExperimentData:
         """
         return self[indices]
 
-    def store(self, filename: str = None):
+    def store(self, filename: Optional[str] = None):
```
Comment (BernardoFerreira) [inline src/f3dasm/_src/experimentdata/experimentdata.py:301]: Please clarify in the method docstring where the object is actually stored.

**Review by BernardoFerreira — CHANGES_REQUESTED** [review]
Docstrings of methods to save data to files should always be clear about where the data is being stored (directory and file).

Please clarify this in both ExperimentData and ExperimentSample store methods.

**Comment by mpvanderschelling** [comment]
I accidentally removed a comment from you Bernardo

**Review by BernardoFerreira — APPROVED** [review]

**Pull request merged**